Index: git/netplan/cli/utils.py
===================================================================
--- git.orig/netplan/cli/utils.py
+++ git/netplan/cli/utils.py
@@ -25,7 +25,7 @@ NM_SNAP_SERVICE_NAME = 'snap.network-man
 
 
 def get_generator_path():
-    return os.environ.get('NETPLAN_GENERATE_PATH', '/lib/netplan/generate')
+    return os.environ.get('NETPLAN_GENERATE_PATH', '/usr/lib/netplan/generate')
 
 
 def is_nm_snap_enabled():  # pragma: nocover (covered in autopkgtest)
