Index: git/zipkin/CMakeLists.txt
===================================================================
--- git.orig/zipkin/CMakeLists.txt
+++ git/zipkin/CMakeLists.txt
@@ -18,7 +18,7 @@ endif()
 
 if (BUILD_SHARED_LIBS)               
   add_library(zipkin SHARED ${ZIPKIN_SRCS})
-  target_link_libraries(zipkin Threads::Threads ${CURL_LIBRARIES} ${WIN32_LIBRARIES})
+  target_link_libraries(zipkin Threads::Threads curl ${WIN32_LIBRARIES})
   set_target_properties(zipkin PROPERTIES VERSION ${ZIPKIN_VERSION_STRING}
                                           SOVERSION ${ZIPKIN_VERSION_MAJOR})
   install(TARGETS zipkin EXPORT zipkinTgt
@@ -35,7 +35,7 @@ endif()
 if (BUILD_STATIC_LIBS)
   add_library(zipkin-static STATIC ${ZIPKIN_SRCS})
   set_target_properties(zipkin-static PROPERTIES OUTPUT_NAME zipkin)
-  target_link_libraries(zipkin-static Threads::Threads ${CURL_LIBRARIES} ${WIN32_LIBRARIES})
+  target_link_libraries(zipkin-static Threads::Threads curl ${WIN32_LIBRARIES})
   install(TARGETS zipkin-static
           ARCHIVE DESTINATION lib)
 endif()
